components/benefit_sponsors/app/controllers/benefit_sponsors/profiles/employers/employer_profiles_controller.rb

```diff
@@ -109,6 +109,8 @@ def export_census_employees
         end
 
         def show_invoice
+          authorize @employer_profile, :show_invoice?
```

Comment:
you can remove explicit calling here.

`authorize @employer_profile` is all it need.